s3repo/view.py

```diff
@@ -1,24 +1,25 @@
 """View for working with the repositories on S3."""
 
 import os
+from typing import List
 
 from flask import Response
 from flask import render_template
 from flask import request
 from flask.views import View
 
+from s3repo.model import S3AsyncModel
```

Comment:
If we want to pass a type here, we need to create an interface for the model, since `S3AsyncModel` is a concrete implementation